bus-mapping/src/evm/opcodes.rs

```diff
@@ -349,8 +349,8 @@ pub fn gen_begin_tx_ops(state: &mut CircuitInputStateRef) -> Result<ExecStep, Er
     let nonce_prev = state.sdb.get_account(&caller_address).1.nonce.as_u64();
     // Increase caller's nonce when the tx is not invalid
     let nonce = if !state.tx.invalid_tx {
-       // state.sdb.increase_nonce(&caller_address) + 1
-       nonce_prev + 1
+        // state.sdb.increase_nonce(&caller_address) + 1
+        nonce_prev + 1
```

Comment:
If the transaction is valid the nonce in the account state should be increased right? Could you explain more why we shouldn't update the nonce in the account for valid transactions they way it was done before (this code changes the behavior for normal transactions)?

Reply:
Ah never mind, I see the code on the master has been updated to not use `increase_nonce` either.